packages/config-plugins/package.json

```diff
@@ -42,6 +42,7 @@
     "fs-extra": "9.0.0",
     "getenv": "^1.0.0",
     "glob": "7.1.6",
+    "gradle-to-js": "^2.0.0",
```

Comment:
Keep the [net size increase](https://packagephobia.com/result?p=gradle-to-js) in mind, this doesn't appear to have much in common with the concept of "config plugins" afaict. This increase would effect packages like expo-constants, react-native-nfc-manager, etc.

Reply:
I have always considered config plugins as a package that has all the utils for interacting with native code, I understand that size of that package is very important, but we don't currently have any other good place to keep it.

I'm happy to move that to eas-cli, it would be even easier for us because we wouldn't have to publish expo-cli, but I also worry that it will lead to a bunch of duplicated code in the future if we go this way

> this doesn't appear to have much in common with the concept of "config plugins" afaict.

I would argue that it's a similar situation to the `ios/utils/Xcodeproj.ts`(but read only, there is no way to serialize back parsed gradle file), we just don't use it yet in plugins. If I understand correctly, this should be here if we wanted to support different flavors for `expo run:android` or provide parsed gradle file for plugins


Either way, I'm fine with moving that to eas-cli, it does not make much difference on our side, so I'm leaving that decision to you. What do you think?